### DISCUSSION TITLE
改的什么名字啊，投票吗？

### DISCUSSION THREAD

**Discussion by pynixwang** [Xiaokang2022/maliang#56] [Q&A] [answered]
tkui?

改名了大板本不到会1.0吗？

**Comment by Xiaokang2022** [comment, marked as answer]
名字将于下个版本发布时说明，版本号顺延，即 v3.0.0，投票只是关于是否更改名称这一问题，至于新名称已经确定，在发布前不会公布新名称，防止出现 PyPI 抢注名称的的情况。

**Reply by pynixwang** [reply]
好吧

**Reply by Xiaokang2022** [reply]
别急，预计下周发布最后一个预览版本：3.0.0rc6

现在 tkintertools 主仓库的任务已经基本完成了，目前的工作是适配三个扩展包，毕竟这个版本的 API 几乎全部改了（以前的奇葩命名太多了），需要把扩展包的 API 都更新一下，此外，演示仓库的 demo 也需要更新，最后文档仓库的文档也需要更新，完成这几件事情，就是正式版了。

预计春节前发布正式版 :)

**Reply by Xiaokang2022** [reply]
你现在看到的主仓库 main 分支就基本是正式版的代码了，除了还有很多 docstring 没来得及改之外（但这问题不大，docstring 不影响代码的运行），功能上只剩下导入外部主题的功能没有写了（之前有，这个版本重写了样式处理的功能，导入外部主题的功能就暂时删去了）

**Reply by pynixwang** [reply]
哈哈哈，我可能不咋用得到，就是在bilibili看到了，过来看看。
